Core/include/Acts/Material/MaterialSlab.hpp

```diff
@@ -79,6 +83,17 @@ class MaterialSlab {
   float m_thicknessInX0 = 0.0f;
   float m_thicknessInL0 = 0.0f;
 
+  static constexpr auto eps = 2 * std::numeric_limits<float>::epsilon();
+
+  constexpr MaterialSlab(const Material& material, float thickness,
+                         [[maybe_unused]] bool dummy)
```

Comment:
What's this used for?

Reply:
unrelated change sorry

Reply:
ah so this is actually necessary so we can have `constexpr` factories like `static constexpr MaterialSlab Nothing()`